### ISSUE TITLE
Make sure paths to assets in Oxygen plugin are local

### ISSUE THREAD

**Issue by raffazizzi** [TEIC/Stylesheets#412]
This may require restructuring the release package.

This issue originated from #312, which was fixed in PR https://github.com/TEIC/Stylesheets/pull/394 by pointing at assets online.

We should make sure that the package can be used offline as well, however.

**Comment by martindholmes** [comment]
Should this be a ticket on the oxygen-tei repo?

**Comment by sydb** [comment]
No, @martindholmes, I think the main point of this ticket is to re-arrange how assets (in particular, CSS files) are stored in the build outputs so that relative paths can be used rather than requiring an internet connection.
Quite possible that the oxygen-tei framework would need a change to deal with a change to the directory structure, too, though.
